gulpfile.js

```diff
@@ -325,7 +325,6 @@ function compile(watch, shouldMinify, opt_preventRemoveAndMakeDir,
           watch: watch,
           preventRemoveAndMakeDir: opt_preventRemoveAndMakeDir,
           minify: shouldMinify,
-          wrapper: '<%= contents %>'
```

Comment:
What does this do exactly? Do you have a snippet of the resulting binary?

Reply:
Asking because of this: https://github.com/ampproject/amphtml/blob/master/gulpfile.js#L826

Reply:
The code path for [minified binaries](https://github.com/ampproject/amphtml/blob/master/gulpfile.js#L804) calls [build-system/tasks/compile.js#closureCompile](https://github.com/cvializ/amphtml/blob/master/build-system/tasks/compile.js#L110) which uses an IIFE as the default wrapper.

Reply:
PTAL

Reply:
Perfect. LGTM from me.